.github/pull_request_template.md

```diff
@@ -18,9 +18,9 @@ How did you make sure this worked? How can a reviewer verify this?
 ```[tasklist]
 # To-do list
 - [ ] Ensure docs build, unit & integration tests, and test coverage pass locally with `make pytest-coverage` (otherwise the merge queue may reject your PR)
-- [ ] For significant ETL changes, ensure the full ETL runs locally
-- [ ] For major data coverage & analysis changes, [run data validation tests](https://catalystcoop-pudl.readthedocs.io/en/latest/dev/testing.html#data-validation)
 - [ ] If updating analyses or data processing functions: make sure to update or write data validation tests
 - [ ] Update the [release notes](../docs/release_notes.rst): reference the PR and related issues.
 - [ ] Review the PR yourself and call out any questions or issues you have
+- [ ] For significant ETL changes, once `make-pytest-coverage` passes, ensure the full ETL runs locally. If your computer can't run it locally, run the `build-deploy-pudl` GitHub Action (or ask someone else with permissions to do so on your behalf). This will also run the validation tests.
+- [ ] For major data coverage & analysis changes, [run data validation tests] (https://catalystcoop-pudl.readthedocs.io/en/latest/dev/testing.html#data-validation) on a complete PUDL SQLite database using `make pytest-validate`, or run the `build-deploy-pudl` GitHub action on your branch (or ask someone else with permissions to help). If running the action, check the `#pudl-deployments` Slack channel or [builds.catalyst.coop] (builds.catalyst.coop) to find the logs for the validation tests and ensure all tests passed.
```

Comment:
This feels too similar to the previous bullet. Maybe we should just lump them together and encourage people to either run the build and review the validation tests or if it's a really small addition just run the pytest coverage and then check the automated nightly builds.